docs/user/getting-startedv2.rst

```diff
@@ -0,0 +1,116 @@
+
+.. _fig-watertankexample:
+.. figure:: images/wt_example/wt-example.png
+    :align: center
+
+    Water Tank Example
+
+The corresponding MaBL code, including descriptive comments, to execute this co-simulation is presented below:
+
+.. literalinclude:: images/wt_example/wt-example.mabl
+    :language: c
+
+To execute this (one can use the file corresponding wt-example.mabl from the unzipped example resouces) run the following command from the terminal:
+
+.. code-block:: none
+
+    > java -jar maestro-2.x.x-jar-with-dependencies.jar --interpret wt-example.mabl
+        where --interpret is to interpret a specification
+
+The result is available in `outputs.csv`. This can be plotted with `pythoncsvplotter.py`, and the result should look like the figure below.
+
+.. _fig-watertankexample-result:
+.. figure:: images/wt_example/wt-example-result.png
+    :align: center
+
+    Result from co-simulation
+
+Part 2: Specification Generation and Expansion
+----------------------------------------------
+This part also concerns the water tank but uses the capabilities of specification generation and expansion to create the specification. Thus, the user does not have to write MaBL by hand. Specification generation and Expansion are not treated in detail in this guide, but more information is available in :ref:`Specification Generation` and :ref:`Expansion`.
+
+The specification generator is based on JSON configuration files with the same structure as the ones used in Maestro1. For this reason, it is also possible to use the `INTO-CPS Application <https://into-cps-association.readthedocs.io/projects/desktop-application/en/latest/>` to create the specification files.
+
+Configuration file
+^^^^^^^^^^^^^^^^^^
+The content of the configuration file is briefly described after the example below. A more detailed description is available at :ref:`sec:legacy_config_format`. The configuration for this example is:
+
+.. literalinclude:: images/wt_example/wt-example-config.json
+   :language: json
+
+- :code:`fmus` contains mappings between a key, enclosed in :code:`{}` and the corresponding FMU.
+- :code:`connections` contains mappings from a single FMU output port to one or more FMU input ports. The format of a port is :code:`{fmuKey}.instance.scalarVariableName`
+- :code:`parameters` contains mappings from a single FMU parameter port and a value.
+- :code:`algorithm` describes that the fixed-step algorithm should be used with the size 0.001. Currently only fixed-step is supported.
+- :code:`end_time` is the end-time of the simulation
+
+Specification Generation
+^^^^^^^^^^^^^^^^^^^^^^^^
+The command below generates multiple specifications at different stages of expansion based on this configuration file:
+
+.. code-block:: none
+
+    > java -jar maestro-2.x.x-jar-with-dependencies.jar --spec-generate1 wt-example-config.json --dump-intermediate "./part2mabl-intermediate" --dump "./part2mabl-specification"
+        where
```

Comment:
same I could like the command to be clean not having additional description in the same "box"